core/src/main/java/com/emc/ia/sdk/sip/client/rest/InfoArchiveRestClient.java

```diff
@@ -207,4 +213,31 @@ private String getValidJsonRequestForExport(String exportConfigurationUri, List<
     return root.toString();
   }
 
+  @Override
+  public LinkContainer uploadTransformationFile(ExportTransformation exportTransformation, File zipFile) throws IOException {
+    String uri = exportTransformation.getUri(LINK_EXPORT_TRANSFORMATION_ZIP);
+    try (InputStream transformationZipStream = new FileInputStream(zipFile)) {
+      checkZipFile(zipFile);
+      Part data = new BinaryPart(zipFile.getName(), "multipart/form-data", transformationZipStream, zipFile.getName());
+      return restClient.post(uri, LinkContainer.class, data);
+    } catch (FileNotFoundException e) {
+      throw new IOException("Zip file with stylesheet doesn't exist", e);
+    }
+  }
+
+  private void checkZipFile(File zipFile) throws FileNotFoundException {
+    if (!zipFile.isFile()) {
+      throw new IllegalArgumentException("Expected file, but passed directory");
+    }
+    String ext;
+    try {
+      ext = zipFile.getName().substring(zipFile.getName().lastIndexOf("."));
```

Comment:
The file name not having a `.` is not an exceptional case and should therefore not be handled with exceptions.